### PR TITLE
Fixes stop animation not working with legacy state animation

### PR DESCRIPTION
**Pull request by abchatra**
Simple fix

### CODE REVIEW

**Comment by abchatra** [comment]
Not sure fix is clean, but works.

**Comment by abchatra** [comment]
Fixes https://github.com/microsoft/pxt-arcade/issues/3278

File: libs/game/animation.ts [hunk 1]
```diff
@@ -567,6 +567,10 @@ namespace animation {
                 return true;
             });
         }
+        if (type == AnimationTypes.All) {
```
Comment (jwunderl) [inline libs/game/animation.ts:570]:
```suggestion
        if (type == AnimationTypes.All || type == AnimationTypes.ImageAnimation) {
```

Feels like it should also probably apply when you pass `ImageAnimation`, as the options are movement vs picture changing?

**Review by jwunderl — APPROVED** [review]
Yeah this is the right fix ~ LGTM either way but I'd lean towards taking the suggestion for consistency~

File: libs/game/animation.ts (hunk 1 above)
Comment (abchatra) [reply libs/game/animation.ts:570]: Yes that would be better. Thanks!

**Pull request merged**